### PR TITLE
Add kube-router as networking solution

### PR DESCRIPTION
**Pull request by murali-reddy**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5071)

### CODE REVIEW

File: docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -168,6 +168,10 @@ sysctl net.ipv4.ip_forward=1
 The result of all this is that all `Pods` can reach each other and can egress
 traffic to the internet.
 
+### Kube-router
+
+[Kube-router](https://github.com/cloudnativelabs/kube-router) is purpose built networking solution for Kubernetes which aims to provide high performance and operational simplicity. Kube-router provides Linux [LVS/IPVS](http://www.linuxvirtualserver.org/software/ipvs.html) based service proxy, Linux kernel forwarding based pod-to-pod networking solution with no overlays and iptables/ipset based network policy enforcer.
```
Comment (zacharysarah) [inline docs/concepts/cluster-administration/networking.md]: Replace with:
```
[Kube-router](https://github.com/cloudnativelabs/kube-router) is a purpose-built networking solution for Kubernetes that aims to provide high performance and operational simplicity. Kube-router provides a Linux [LVS/IPVS](http://www.linuxvirtualserver.org/software/ipvs.html)-based service proxy, a Linux kernel forwarding-based pod-to-pod networking solution with no overlays, and iptables/ipset-based network policy enforcer. 
```

**Comment by zacharysarah** [comment]
@Bradamant3 :wave: Would you mind taking over this PR while I'm out on vacation? If follow-up edits look good, feel free to dismiss my review. 🙇

**Comment by murali-reddy** [comment]
Have updated as per @zacharysarah suggestion.

**Comment by zacharysarah** [comment]
LGTM 👍 

@Bradamant3: Never mind! Thanks!

**Pull request merged**